srcpkgs/gpgme/template

```diff
@@ -1,13 +1,15 @@
 # Template file for 'gpgme'
 pkgname=gpgme
 version=1.23.1
-revision=1
+revision=2
 build_style=gnu-configure
+build_helper="python3"
 configure_args="--enable-fd-passing
  --with-libgpg-error-prefix=$XBPS_CROSS_BASE/usr
  --with-libassuan-prefix=$XBPS_CROSS_BASE/usr"
-hostmakedepends="gnupg pkg-config qt5-host-tools qt5-qmake"
-makedepends="libassuan-devel qt5-tools-devel"
+hostmakedepends="gnupg pkg-config qt5-host-tools qt5-qmake python3-setuptools python3-installer python3-build python3-wheel swig python3-devel"
```

Comment:
wrap at 80 chars and python3-devel isn't needed on host

```suggestion
hostmakedepends="gnupg pkg-config qt5-host-tools qt5-qmake python3-setuptools
 python3-installer python3-build python3-wheel swig"
```

Reply:
Done